### ISSUE TITLE
Package Dependencies

### ISSUE THREAD

**Issue by chrisyoyo** [httpswift/swifter#168]
When I create my package with: swift package init --type executable
and i add to my Package.swift ->

dependencies: [
        .Package(url: "https://github.com/httpswift/swifter.git", majorVersion: 1, minor: 2)
  ])

And then I try to "swift build" it appears a lot of errors... 

One of them:

/Users/yovano_c/Desktop/GameServer/Packages/Swifter-1.2.6/Sources/File.swift:109:34: error: 'inout' before a parameter name is not allowed, place it before the parameter type instead
    public func read(inout data: [UInt8]) throws -> Int {
                     ~~~~~       ^
                                 inout

Why can't we use this package like others?...

**Comment by glock45** [comment]
SPM was supported only by the stable branch and now it should work for the master (Swift 3). I've just added git tag. Please have a look at the sample app below.

> 
kolakowski$ export PATH=/Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin:"${PATH}**"
damiankolakowski$ which swift
/Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift
kolakowski$ swiftc --version
Apple Swift version 3.0 (swiftlang-800.0.41.2 clang-800.0.36)
Target: x86_64-apple-macosx10.9
kolakowski$ swift build --version
Apple Swift Package Manager - Swift 3.0 (swiftpm-15.2)
kolakowski$ swift build
Cloning https://github.com/httpswift/swifter.git
HEAD is now at bf4121a Fixed SPM build.
Resolved version: 3.0.0
Compile CSQLite sqlite.c
Linking CSQLite
Compile Swift Module 'Swifter' (19 sources)
Compile Swift Module 'Test_SPM_Swifter' (1 sources)
Linking .build/debug/Test SPM Swifter
kolakowski$ "./.build/debug/Test SPM Swifter"
Server runs on 9080...

Here is my sample app: [app.zip](https://github.com/httpswift/swifter/files/394850/Archive.zip)

**Comment by glock45** [comment]
After emails exchange I can close this.
